_docs/setup/kubernetes/multicluster-install.md

```diff
@@ -0,0 +1,169 @@
+on **one** Kubernetes cluster.  Istio must be [deployed with Helm]({{home}}/docs/setup/kubernetes/helm-install.html) using the `--set global.multicluster.enabled=true` flag.  This is currently mandatory.
+
+* Currently only [manual sidecar injection]({{home}}/docs/setup/kubernetes/sidecar-injection.html#manual-sidecar-injection)
+has been validated with multicluster.
+
+## Install a multicluster configmap on the Istio control plane
+
+> Pilot will not start until the configmap in these instructions
+is created.  This is normal behavior but different from what is seen with
+multicluster disabled.
+
+Create a `clusterregistry.k8s.io` descriptor to describe each Kubernetes cluster's role
+in the multicluster environment.  These files should be stored in the directory
+`$HOME/multicluster`, have a `.yaml` extension, and have a unique filename.
+
+An example `clusterregistry.k8s.io` configuration descriptor is shown below:
+
+```yaml
+apiVersion: clusterregistry.k8s.io/v1alpha1
+kind: Cluster
+metadata:
+  name: "falkor07"
+  annotations:
+    config.istio.io/pilotEndpoint: "192.168.1.1:9080"
+    config.istio.io/platform: "Kubernetes"
+    config.istio.io/pilotCfgStore: True
+    config.istio.io/accessConfigFile: "falkor07.kube.conf"
+  spec:
+    kubernetesApiEndpoints:
+      serverEndpoints:
+        - clientCIDR: "10.23.230.0/28"
+          serverAddress: "10.23.230.11"
+```
+
+A unique security context is required to describe how to securely access
+each Kubernetes cluster in the system.  In the above example,
+`falkor07.kube.conf` is the Istio control plane credential file. The
+`config.istio.io/pilotCfgStore: True` flag is set in the condition. This
+`clusterregistry.k8s.io` descriptor describes the Istio control plane Kubernetes
+cluster.  For remotes, please set `pilotCfgStore: False`.
+
+> The implementation only uses the
+`config.istio.io/pilotCfgStore` and `config.istio.io/accessConfigFile`
+annotations, although every other annotation and spec is validated for
+correct syntax.  They may be set to dummy values, as long as they are
+syntactically correct.
+
+If the prerequisites are met, the credentials for each Kubernetes cluster
+will also be present in `$HOME/multicluster`.
+
+Assemble and create the configmap from these files in the Kubernetes cluster
+where the Istio control plane is operational:
+
+```bash
+kubectl create configmap -n istio-system multicluster -f $HOME/multicluster
+```
+
+## Deploy on the remote cluster
+
+Multicluster makes **one** Istio service mesh from all Kubernetes clusters
+that participate in multicluster.  Multicluster uses **one** Kubernetes
+cluster to run the Istio control plane.  **Any reasonable** number of remote
+Kubernetes clusters may be attached to the **one** Istio control plane.
+
+### Use kubectl with Helm to connect the remote cluster
+
+1. Use the helm template command on a remote to specify the Istio control plane:
```

Comment:
for istio-remote, does user need to deploy it to each of the remote k8s clusters?  This was not clear to me.

Reply:
clarified.